proto/sequencers/tx.proto

```diff
@@ -1,43 +1,40 @@
 syntax = "proto3";
 package rollapp.sequencers.types;
 
-import "gogoproto/gogo.proto";
 import "cosmos/msg/v1/msg.proto";
 import "google/protobuf/any.proto";
 
 option go_package = "github.com/dymensionxyz/dymension-rdk/x/sequencers/types";
 
 // Msg defines the Msg service.
 service Msg {
-    rpc CreateSequencer(MsgCreateSequencer) returns (MsgCreateSequencerResponse);
-    rpc UpdateSequencer(MsgUpdateSequencer) returns (MsgUpdateSequencerResponse);
+    // Messages
```

Comment:
ain't the `Update...` msgs also expected to be `Consensus Messages`?

Reply:
it's just regular msgs . consensusMsg is a wrapper which can bypass the tx signature by the sequencer. 
some msg handlers may or may not be also eligble to be called directly from the l2.

Reply:
didn't we discussed that all the updates will go through the consensus based msgs? so it will be updated only on the L1
or it will be changed later?